tests/models/test_parameterized_query.py

```diff
@@ -45,12 +45,94 @@ def test_handles_objects(self):
         })
         self.assertEqual(set([]), query.missing_params)
 
-    def test_raises_on_parameters_not_in_schema(self):
+    def test_single_invalid_parameter_exception(self):
+        query = ParameterizedQuery("foo")
+        with pytest.raises(InvalidParameterError) as excinfo:
+            query.apply({"bar": None})
+        
+        message, parameter_errors = excinfo.value.args
+        self.assertEquals(message, 'Parameter "bar" is invalid.')
+        self.assertEquals(len(parameter_errors), 1)
+
+    def test_multiple_invalid_parameter_exception(self):
+        query = ParameterizedQuery("foo")
+        with pytest.raises(InvalidParameterError) as excinfo:
+            query.apply({"bar": None, "baz": None})
+        
+        message, parameter_errors = excinfo.value.args
+        self.assertEquals(message, 'Parameters "bar", "baz" are invalid.')
```

Comment:
This is a flaky test cause it may come out `"bar", "baz"` or `"baz", "bar"` depending on the all-mighty.
@rauchy @arikfr any ideas how to mitigate this?

Reply:
I'll sort by alphabet for now. Lmk if I should use an `OrderedDict` instead or sth.